pkg/server/handler/v1alpha1/integration.go

```diff
@@ -13,11 +13,12 @@ import (
 	"k8s.io/client-go/util/retry"
 
 	"github.com/caicloud/cyclone/pkg/apis/cyclone/v1alpha1"
+	"github.com/caicloud/cyclone/pkg/common"
```

Comment:
If we move `NewClusterClient()` from `pkg/common` into `pkg/util`, we can avoid a lot of codes for package name conflict in this PR.